packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -89,91 +304,48 @@ def create_client(
     built_in_import_modules: List[str] = []
     custom_import_modules: List[str] = []
```

Comment:
Can we create these variables in the respective methods, where it is used and pass it  to other functions from there. Because, we are just passing it to `_create_client`.